consensus/src/state_computer.rs

```diff
@@ -148,7 +172,41 @@ impl StateComputer for ExecutionProxy {
             .await
             .expect("Failed to send async state sync notification");
 
-        Ok(())
+        // If there are no blocks, epoch and round will be invalid.
+        // TODO: is this ever the case? why?
```

Comment:
Any idea if it's expected that commit will always have at least one block? Seems like it should, but I'm not sure at all.

Reply:
yeah it should have at least one block